### PR TITLE
[artifact-manager,vra-ng] (#207) Remove regional content support from `vra-ng` projects

### PR DESCRIPTION
**Pull request by Michaelpalacce**
### Checklist

- [x] I have added relevant error handling and logging messages to help troubleshooting
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation, relevant usage information (if applicable)
- [x] I have updated the PR title with affected component, related issue number and a short summary of the changes introduced
- [x] I have added labels for implementation kind (kind/*) and version type (version/*)
- [x] I have tested against live environment, if applicable
- [x] I have synced any structure and/or content vRA-NG improvements with vra-ng and ts-vra-ng archetypes (if applicable)
- [x] I have my changes rebased and squashed to the minimal number of relevant commits. **Notice: don't squash all commits**
- [x] I have added a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### Release Notes

Removed Regional Content from everywhere it was used

### Related issues and PRs
#207

### CODE REVIEW

File: docs/versions/latest/Release.md [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
 [//]: # (Features -> New Functionality)
 
+### *Deprecated Regional Content*
+
+The Regional content supported in previous versions of BTVA has been removed. Unfortunately that part of the build tools never functioned the way we wanted it to, and managing the "infrastructure" tab in Assembler is no longer something we want to do as it contradicts the principles behind the `vra-ng` archetype.
```
Comment (VenelinBakalov) [inline docs/versions/latest/Release.md:22]:
```suggestion
The Regional content supported in previous versions of Build Tools for Aria has been removed. Unfortunately that part of the build tools never functioned the way we wanted it to, and managing the "infrastructure" tab in Assembler is no longer something we want to do as it contradicts the principles behind the `vra-ng` archetype.
```

**Comment by VenelinBakalov** [comment]
Do we need to update anything in the installer?

File: docs/versions/latest/Components/Archetypes/vRA 8.x/General/Known Issues.md [hunk 2]
```diff
@@ -4,9 +4,7 @@ A list of known vRA 8.x archetype issues
 
 ## Table Of Contents
 
-1. [Custom Resources Known Issues](../Components/Custom%20Resources.md#known-issues)
```
Comment (VenelinBakalov) [inline docs/versions/latest/Components/Archetypes/vRA 8.x/General/Known Issues.md:7]: Why are we removing those 2 references?

Comment (VenelinBakalov) [reply docs/versions/latest/Components/Archetypes/vRA 8.x/General/Known Issues.md:7]: Hmmm..or they are not existing to begin with..

**Review by VenelinBakalov — CHANGES_REQUESTED** [review]
just 1 minor comment left

File: docs/versions/latest/Components/Archetypes/vRA 8.x/General/Known Issues.md (hunk 2 above)
Comment (Michaelpalacce) [reply docs/versions/latest/Components/Archetypes/vRA 8.x/General/Known Issues.md:7]: Yeah 😄

**Comment by Michaelpalacce** [comment]
@VenelinBakalov done! Syncing with main now if needed

**Comment by Michaelpalacce** [comment]
Turns out no need to sync

**Review by VenelinBakalov — APPROVED** [review]

**Pull request merged**